docker/unit-tests.test.yml

```diff
@@ -1,10 +1,10 @@
 version: '3.4'
 
 services:
-    sut:
-        build:
-            context: ..
-            dockerfile: docker/Dockerfile
-            target: builder
-        working_dir: /usr/local/go/src/github.com/go-debos/debos/actions
-        command: go test
+  debos-unit-tests:
```

Comment:
Will this rename still trigger testing inside dockerhub?

Reply:
no, this breaks it. whoops. fixed now